var/spack/repos/builtin/packages/libxpm/package.py

```diff
@@ -26,6 +26,6 @@ class Libxpm(AutotoolsPackage):
     depends_on('pkgconfig', type='build')
     depends_on('util-macros', type='build')
 
-    def setup_environment(self, spack_env, run_env):
-        spack_env.append_flags('LDFLAGS', '-L{0} -lintl'.format(
-            self.spec['gettext'].prefix.lib))
+#    def setup_environment(self, spack_env, run_env):
```

Comment:
My error is due to these three lines being commented out.  Was this intentional?

Reply:
Err... Sorry I don't know how that slipped in. I will submit a PR to fix this ASAP.

Reply:
Thanks!